lib/recipe/widgets/ingredient_form.dart

```diff
@@ -141,6 +139,35 @@ class IngredientForm extends StatelessWidget {
                               recipe: state.recipes[recipeIndex])),
                       selectedItem: state.recipes[recipeIndex]
                           .ingredients[ingredientIndex].cookingState),
+                  Visibility(
+                    visible: (state.recipes[recipeIndex]
+                            .ingredients[ingredientIndex].cookingState ==
+                        "Other (please specify)"),
+                    child: TextFormField(
+                      initialValue: state.recipes[recipeIndex]
+                          .ingredients[ingredientIndex].customCookingState,
+                      decoration: InputDecoration(
+                        icon: const Icon(Icons.set_meal_rounded),
+                        labelText: 'Specify cooking state',
+                        helperText: 'Cooking state e.g. Barbequed',
+                        errorText: (state.recipes[recipeIndex]
+                                    .ingredients[ingredientIndex].customName ==
```

Comment:
I missed this bug in review (`customName` should be `customCookingState`) — fixed it in #50.